modules/ipi-install-network-requirements.adoc

```diff
@@ -70,6 +90,13 @@ The following table provides an exemplary embodiment of hostnames for each node
 | Worker-n | openshift-worker-n.<cluster-name>.<domain> | <ip>
 |=====
 
+ifdef::upstream[]
+For assistance in configuring the DHCP server, check xref:ipi-install-upstream-appendix[Appendix] section for:
```

Comment:
No xrefs allowed in modules

Reply:
It's in a conditional tag for upstream, and won't appear downstream.